templates/rootkitHunter/rootkitHunter.cfg

```diff
@@ -0,0 +1,53 @@
+PATROL_CONFIG
+"/SENTRY/X/rootkitHunter/actionList" = { REPLACE = "" },
+"/SENTRY/X/rootkitHunter/collectionErrorCountAutoAcknowledgeTime" = { REPLACE = "900" },
+"/SENTRY/X/rootkitHunter/credentialsList" = { REPLACE = "" },
+"/SENTRY/X/rootkitHunter/databaseType" = { REPLACE = "" },
+"/SENTRY/X/rootkitHunter/description" = { REPLACE = "Monitor the Rootkit===================Scan the Rootkit for malware.Setup-----Add the Linux host to be monitored with credentials to execute the rkhunter command and activate the template." },
+"/SENTRY/X/rootkitHunter/disabled" = { REPLACE = "0" },
+"/SENTRY/X/rootkitHunter/embeddedFileList" = { REPLACE = "" },
+"/SENTRY/X/rootkitHunter/label" = { REPLACE = "Rootkit Hunter" },
+"/SENTRY/X/rootkitHunter/macroList" = { REPLACE = "" },
+"/SENTRY/X/rootkitHunter/messageList" = { REPLACE = "" },
+"/SENTRY/X/rootkitHunter/parameterList" = { REPLACE = "CollectionErrorCount;1" },
+"/SENTRY/X/rootkitHunter/productVersion" = { REPLACE = "10300" },
+"/SENTRY/X/rootkitHunter/runAlertForEachCollectionError" = { REPLACE = "1" },
+"/SENTRY/X/rootkitHunter/shareCookies" = { REPLACE = "1" },
+"/SENTRY/X/rootkitHunter/systemType" = { REPLACE = "unix" },
+"/SENTRY/X/rootkitHunter/thresholdList" = { REPLACE = "CollectionErrorCount;0 0 0 0 0 0;1 1 2147483646 0 0 2;0 0 0 0 0 0" },
+"/SENTRY/X/rootkitHunter/type" = { REPLACE = "Template" },
+"/SENTRY/X/rootkitHunter:commandRKhunterCUpdateSk/actionList" = { REPLACE = "" },
+"/SENTRY/X/rootkitHunter:commandRKhunterCUpdateSk/credentials" = { REPLACE = "System Credentials" },
+"/SENTRY/X/rootkitHunter:commandRKhunterCUpdateSk/exitCodeStatus" = { REPLACE = "0" },
+"/SENTRY/X/rootkitHunter:commandRKhunterCUpdateSk/exitCodes" = { REPLACE = "0" },
+"/SENTRY/X/rootkitHunter:commandRKhunterCUpdateSk/label" = { REPLACE = "RKhunter Check" },
+"/SENTRY/X/rootkitHunter:commandRKhunterCUpdateSk/messageList" = { REPLACE = "" },
+"/SENTRY/X/rootkitHunter:commandRKhunterCUpdateSk/neverEnding" = { REPLACE = "0" },
+"/SENTRY/X/rootkitHunter:commandRKhunterCUpdateSk/osCommand" = { REPLACE = "rkhunter -c  --sk --rwo;rm /var/log/rkhunter/rkhunter.log" },
```

Comment:
Rather than deleting `rkhunter`'s log file, isn't there an option to make it log to a different directory (like in **/tmp**) and then we delete this temporary file? Because if a sysadmin uses `rkhunter` and its logs are deleted twice a day, that may be annoying.

Reply:
Modified the command to write to /tmp